exercises/K-functions-parameters/exercise4.js

```diff
@@ -1,5 +1,7 @@
 // Declare your function first
-
+function sum(num1,num2){
+    return num1+num2
+}
 // Call the function and assign to a variable `sum`
 
-console.log(sum);
+console.log(sum(13,124));
```

Comment:
This will work, but in the exercise it mentioned to assign the value to a variable called sum, so:
`let sum = add(13, 124)
console.log(sum)`